ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -54,20 +54,23 @@ const ProjectOverview = (props) => {
           }
           {props.project.canEdit ? <span><br /><EditFamiliesAndIndividualsButton /></span> : null }<br />
         </div>
+        {/*
```

Comment:
I'd rather this was deleted than commented out. These kinds of comments actually render to the DOM